packages/apollo-server-module-graphiql/src/renderGraphiQL.ts

```diff
@@ -30,6 +30,7 @@ export type GraphiQLData = {
   passHeader?: string;
   editorTheme?: string;
   websocketConnectionParams?: Object;
+  rewriteURL?: boolean;
```

Comment:
Maybe make this `rewriteURL: boolean = false` so it's clearer what the default value is if you don't pass this option?